language-guides/spring-data-neo4j/spring-data-neo4j.adoc

```diff
@@ -34,129 +34,206 @@ When developing with Neo4j, please use JDK 8 and your favorite IDE.
-Additionally it provides support for Spring Conversions, Transaction Handling, Spring-Data Repositories, Spring-Data REST and Spring-Boot.
+Spring Data Neo4j integrates the link:../neo4j-ogm[Neo4j-OGM] library to provide fast and comprehensive object-graph mapping.
+It also provides support for Spring conversions, transaction handling, Spring Data repositories, Spring Data REST, and Spring Boot.
 
-To get started, add Spring Data Neo4j as a dependency, then configure the necessary beans in your Java config.
-Then you can annotate (optional) your entities and define Spring-Data-Repositories as convenient interfaces to your persistence layer.
+To use Spring Data Neo4j, all you need is the Spring Data Neo4j and OGM dependencies and a few annotations to do the object-graph mapping.
+Then, you can annotate your entities and define Spring-Data-Repositories as convenient interfaces to your persistence layer.
 
 ==== Features
 
-* Spring Boot Integration (since 1.4)
-* annotation based object graph mapping
-* interface based repository support with annotated and derived finder methods
+* Spring Boot Integration
+* annotation-based object-graph mapping
+* interface-based repository support with annotated and derived finder methods
 * fast class metadata scanning
 * optimized management of data loading and change tracking for minimal data transfers
-* multiple transports: *binary protocol*, HTTP and embedded
+* multiple transports: *binary protocol*, HTTP, and embedded
 * persistence lifecycle events
 
 ==== Quickstart
 
-.Spring-Data-Neo4j Dependency
+Spring Boot takes on much of the responsibility of application configuration and bootstrap, so we have chosen to take advantage of that assistance in our project, as well.
+The example project code for https://github.com/neo4j-examples/movies-java-spring-data-neo4j[Spring Data Neo4j] is checked in to GitHub.
+You can clone the repository and run the code along with this guide, or you can build the project from the ground up from the http://start.spring.io[Spring Initializr] page.
+For this approach, you can follow along with a <link>[blog post] for step-by-step instructions.
+
+Dependencies for Spring Data Neo4j and OGM need to be added in order to use the capabilities.
+The latest OGM version requires a Neo4j version one release back, so we have added a dependency for Neo4j 3.2.9 specifically.
+The rest of the dependencies set up the object-graph mapper so that we can relate our POJOs (plain old java objects) to the graph backend.
+
+.Spring-Data-Neo4j, Spring Boot, OGM Dependencies
 [source,xml,subs="verbatim,attributes"]
 ----
 <dependencies>
     <dependency>
-        <groupId>org.springframework.data</groupId>
-        <artifactId>spring-data-neo4j</artifactId>
-        <version>{spring-data-neo4j-version}</version>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-data-neo4j</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
but you have to tell Spring Boot to use Spring Data with the embedded Neo4j database, correct? It would be just like using H2. You have to add a dependency for H2, if you used that. You have to tell it what to use as backend, if I understand your question correctly.....

Reply:
For the server mode (and without tests) it is not necessary to define the dependency. It is just needed if we plan to provide tests and/or want to use embedded.